app/src/main/java/ch/epfl/culturequest/ui/profile/ProfileFragment.java

```diff
@@ -32,12 +32,19 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         binding = FragmentProfileBinding.inflate(inflater, container, false);
         View root = binding.getRoot();
 
-        final TextView textView = binding.profileName;
+        final TextView profileName = binding.profileName;
+        final TextView profilePlace = binding.profilePlace;
         final CircleImageView profilePicture = binding.profilePicture;
         final RecyclerView pictureGrid = binding.pictureGrid;
 
+        FollowButton followButton = new FollowButton(binding.profileFollowButton);
+        profileViewModel.getFollowed().observe(getViewLifecycleOwner(), followButton::setFollowed);
+        followButton.setOnClickListener(v -> profileViewModel.changeFollow());
 
-        profileViewModel.getName().observe(getViewLifecycleOwner(), textView::setText);
+        profilePlace.setText("Lausanne");
```

Comment:
Maybe it could be cool to have a check that only displays the follow button if the Profile the user is visiting is not its own

Reply:
This is handled by the new adaptive profile of Lucas so I need to wait to integrate this.